csfieldguide/static/interactives/password-guesser/scss/password-guesser.scss

```diff
@@ -0,0 +1,46 @@
+#pg-database {
+	max-width: 100%;
+	font-family: monospace;
+	.overflow-table {
+		padding: 0px !important;
+		max-width: 100%;
+		white-space: nowrap;
+		overflow: auto;
+	}
+	td, th {
+		padding-left: 20px;
+		padding-right: 20px;
+		border: 1px solid #c7c7c8;
+	}
+	th {
+		background-color: #dde0e2;
+	}
+	tr {
+		background-color: #eceeef;
+	}
+}
+
+#pg-checker, #pg-hints {
+	max-width: 1000px;
+	details {
+		border: 1px solid;
+		border-radius: 5px;
+		padding: 0.5rem;
+	}
+	summary {
+		font-weight: bold;
+		padding: 0.5rem;
+	}
+	details[open] {
+		padding: 0.5rem;
+	}
+}
+
+.highlight {
+	background-color: #28a745 !important;
```

Comment:
Highlight doesn't highlight the whole div on mobile

![image](https://user-images.githubusercontent.com/16066822/66090914-df674480-e5e1-11e9-8ef6-25cf2affbcca.png)